protobuf will fail pkgconf test pipeline

If the pkgconf test pipeline is added to protobuf it will fail in the following way:

```
2024/11/12 23:14:23 INFO running step "test/pkgconf"
2024/11/12 23:14:23 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:14:23 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:14:23 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:14:23 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:14:23 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + basename /home/build/melange-out/protobuf-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + dev_pkg=protobuf-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + apk info -L protobuf-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + grep -q ^Name: usr/lib/pkgconfig/protobuf-lite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + basename usr/lib/pkgconfig/protobuf-lite.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + lib_name=protobuf-lite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + echo usr/lib/pkgconfig/protobuf-lite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + grep -q '^usr/lib/pkgconfig/protobuf-lite.pc$\|^usr/share/pkgconfig/protobuf-lite.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + pkgconf --exists protobuf-lite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN + pkgconf --print-errors --exists protobuf-lite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package absl_absl_check was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Perhaps you should add the directory containing `absl_absl_check.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_absl_check', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_absl_log', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_algorithm', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_base', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_bind_front', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_bits', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_btree', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_cleanup', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_cord', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_core_headers', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_debugging', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_die_if_null', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_dynamic_annotations', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_flags', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_flat_hash_map', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_flat_hash_set', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_function_ref', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_hash', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_if_constexpr', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_layout', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_log_initialize', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_log_globals', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_log_severity', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_memory', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_node_hash_map', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_node_hash_set', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_optional', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_random_distributions', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_random_random', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_span', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_status', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_statusor', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_strings', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_synchronization', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_time', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_type_traits', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_utility', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_variant', required by 'protobuf-lite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 WARN Package 'absl_strings', required by 'utf8_range', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:23 INFO pod 2f3be58a56dfe3736fcc9cf1f450cd1669102942f5fcba6fa6229fa35ced98a1 terminated
2024/11/12 23:14:23 INFO pod 2497ca56028474b7dfa9674cc284ff2cf0fe4f86a7ebf80f6b9f5fdaba30a071 terminated
2024/11/12 23:14:23 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:14:23 INFO   workspace dir: /home/user/tmp/melange-workspace-295163664
2024/11/12 23:14:23 INFO   guest dir: /home/user/tmp/melange-guest-1124104187
2024/11/12 23:14:23 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/protobuf] Error 1
```

--- a/protobuf.yaml
+++ b/protobuf.yaml
@@ -67,12 +67,16 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
-        - protobuf
-        - zlib-dev
-        - protoc
+        - abseil-cpp-dev
         - libprotoc
         - libprotobuf
         - libprotobuf-lite
+        - protobuf
+        - protoc
+        - zlib-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: protoc
     description: Protocol buffer compiler binary and library

